### ISSUE TITLE
unsafe-reset command panics

### ISSUE THREAD

**Issue by mindstyle85** [celestiaorg/celestia-app#466]
## Summary of Bug

The unsafe reset command is panicking after it cant delete/find the atomic file it is expecting to. I have checked our own validator and noticed we also do not have that file generated,. Is it possible the file is no longer used in new SDK and the unsafe command still expects it to be there?

## Version

I didnt get the info from the user, but he used the guide so should be 0.5.2 i guess, but i also experienced this on 0.5.0 when we started.. 

## Steps to Reproduce

```
celestia-appd tendermint unsafe-reset-all --home $HOME/.celestia-app
Removed all blockchain history dir=data
panic: open config/write-file-atomic-7099847883190097162: no such file or directory
```

---

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by evan-forbes** [comment]
thanks for creating an issue @mindstyle85 please see https://github.com/cosmos/cosmos-sdk/issues/12106

it looks like this should be fixed when we pull in changes from v0.35.6 #475

**Comment by evan-forbes** [comment]
I linked it to the PR that bumps tendermint, so it should automatically close after that is merged. We should definitely still check manually tho, we might have to reopen it after the merge
